server/views/partials/breadcrumb.njk

```diff
@@ -1,14 +1,14 @@
 {% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
```

Comment:
note that the govuk breadcrumb component is all lowercase here - see https://design-system.service.gov.uk/components/breadcrumbs/